README.md

```diff
@@ -51,7 +51,7 @@ A convenient alternative is to let the SONiC build system configure a build envi
 
 2. Build the sonic-utilities Python wheel package inside the Bullseye slave container, and tell the build system to keep the container alive when finished
     ```
-    make NOSTRETCH=1 NOBUSTER=1 KEEP_SLAVE_ON=yes target/python-wheels/bullseye/sonic_utilities-1.2-py3-none-any.whl
+    make NOSTRETCH=1 NOBUSTER=1 KEEP_SLAVE_ON=yes target/python-wheels/bookworm/sonic_utilities-1.2-py3-none-any.whl
```

Comment:
`NOSTRETCH=1 NOBUSTER=1` should be replaced with `-f Makefile.work BLDENV=bookworm` (this has the advantage of being clearer that only this target (and dependencies) and this distro, at the risk of going a bit into the internals of the build system).

Reply:
updated as requested.